Dockerfile

```diff
@@ -1,52 +1,95 @@
 FROM fedora:31
 
 # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1694411
```

Comment:
The comment should go away now since it was referring to the line below it and is therefore orphaned. It was also only an issue that I ran only when building for ppc64le, which we don't test for anymore.

Reply:
I removed the comment.